js/index.js

```diff
@@ -1 +1,38 @@
-// Your code goes here
+
+
+let bus = document.querySelector('.logo-heading');
+bus.addEventListener('mouseover', function(){ 
+bus.style.color = 'orange'; 
+});
+
+document.addEventListener('keydown', logKey);
+function logKey(e) {
+log.textContent += ` ${e.code}`;
```

Comment:
At this point, the `log` object does not exist. You can use `querySelector` to add a DOM element or just do a console.log